script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -361,12 +370,13 @@ Did the desire to kill whip your blood into a frenzy, Mr. Murderer?"
 [sha:fb5defe4e1cb192bc34fafb119827210c9a035fc]{
 -- Page 600, Offset 18079.
 -- 人間の尺度で返答する。
-I respond in accordance with my human values.
+// rather than values, this is more something like 'human standards'
```

Comment:
I'm not sure I get the difference? Old version reads more nicely unless there's something I'm too thick to understand